CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### 4.2.2 (Next)
 
 * Your contribution here.
+* [#314](https://github.com/alexa-js/alexa-app/pull/314): Fix breaking logic in request.slot() - [@User1m](https://github.com/user1m)
```

Comment:
My ADD says this is missing a period at the end ;)

Reply:
sounds good.